<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qucs-RFlayout</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="l_input">
      <property name="text">
       <string>Input :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="pb_read">
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QComboBox" name="cb_format"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="l_output">
      <property name="text">
       <string>Output :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="le_path_out"/>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QLabel" name="l_format">
      <property name="text">
       <string>Format :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QPushButton" name="pb_browse_out">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="pb_write">
      <property name="text">
       <string>Write</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="le_path_in"/>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QPushButton" name="pb_browse_in">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2" colspan="2">
     <widget class="Preview" name="glw_preview" native="true"/>
    </item>
    <item row="2" column="2" colspan="4">
     <widget class="QTextBrowser" name="tb_log">
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>867</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Preview</class>
   <extends>QWidget</extends>
   <header>preview.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
